src/view/droppable/connected-droppable.js

```diff
@@ -151,13 +153,20 @@ const makeMapStateToProps = () => {
 // Leaning heavily on the default shallow equality checking
 // that `connect` provides.
 // It avoids needing to do it own within `Droppable`
-// $ExpectError - no idea how to type this correctly
-export default connect(
+const connectedDroppable: OwnProps => Node = (connect(
   // returning a function to ensure each
   // Droppable gets its own selector
-  makeMapStateToProps,
+  (makeMapStateToProps: any),
   null,
   null,
   { storeKey },
-)(Droppable);
+): any)(Droppable);
 
+connectedDroppable.defaultProps = ({
```

Comment:
Sad times! We needed this one?

Reply:
This is more honest. This is the component used outside,  so keeping all props required is incorrect.